students/YingGuo/session03/mailroom.py

```diff
@@ -0,0 +1,65 @@
+#hw3, mailroom
+
+data = {"Tom":[100,200], "Tim":[3000], "Ken": [888,888,888], "Ted":[8000,5000,500], "Jon":[1000,6000]}
```

Comment:
Oh, you are already using a dict! :) I think that's fine, though it hasn't been covered in class yet (this week's topic).